src/components/tracked-controls.js

```diff
@@ -80,8 +91,10 @@ module.exports.Component = registerComponent('tracked-controls', {
 
     // Hand IDs: 0 is right, 1 is left.
     for (i = 0; i < controllers.length; i++) {
-      if ((data.idPrefix && controllers[i].id.indexOf(data.idPrefix) === 0) ||
-          (!data.idPrefix && controllers[i].id === data.id)) {
+      if (((data.idPrefix && controllers[i].id.indexOf(data.idPrefix) === 0) ||
+           (!data.idPrefix && controllers[i].id === data.id)) &&
+          ((data.hand === 'any-or-none') ||
+           ((controllers[i].hand === 'undefined' ? DEFAULT_HANDEDNESS : controllers[i].hand) === (data.hand === 'none' ? '' : data.hand)))) {
```

Comment:
Is this a typo or intentional? In src/utils/tracked-controls.js  you are testing against: `controllers[i].hand === undefined`